config/environments/test.rb

```diff
@@ -8,13 +8,14 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
-  config.cache_classes = false
+  # While tests run files are not watched, reloading is not necessary.
+  config.enable_reloading = false
   config.action_view.cache_template_loading = true
-
-  # Do not eager load code on boot. This avoids loading your whole application
-  # just for the purpose of running a single test. If you are using a tool that
-  # preloads Rails for running tests, you may have to set it to true.
-  config.eager_load = false
+  # Eager loading loads your entire application. When running a single test locally,
+  # this is usually not necessary, and can slow down your test suite. However, it's
+  # recommended that you enable it in continuous integration systems to ensure eager
+  # loading is working properly before deploying your code.
+  config.eager_load = ENV['CI'].present?
```

Comment:
Did we add this variable in travis for aviary-public?

Reply:
No, if its not set then it will not set eager_load = false and if it sets then it will set eager_load = true

We can decide what is preferred and we can add that variable to env

Reply:
My bad, this is one of the default env variables that Travis has, and its true.